JS transforms: Add docs (#466)
ConduitIO/conduit: README.md

Conduit is also able to run standalone connectors. If you are interested in writing a connector yourself, have a look at our [Go Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk). Since standalone connectors communicate with Conduit through gRPC they can be written in virtually any programming language, as long as the connector follows the [Conduit Connector Protocol](https://github.com/ConduitIO/conduit-connector-protocol).

## Transforms

Transforms are operations applied on single records. Transforms are optional parts of a pipeline and, when present, they
come in the form of pipeline components called processors.

> Conduit provides a number of built-in transforms, such as filtering fields, replacing them, posting payloads to HTTP endpoints etc.

Reviewer: What do you think about the following change?

```suggestion
Conduit provides a number of built-in transforms, which can be used to filter and replace fields, post payloads to HTTP endpoints etc.
```

Author: I like it!
